src/openrct2/actions/SetCheatAction.hpp

```diff
@@ -685,14 +685,12 @@ DEFINE_GAME_ACTION(SetCheatAction, GAME_COMMAND_CHEAT, GameActionResult)
             }
         }
 
-        for (spriteIndex = gSpriteListHead[SPRITE_LIST_PEEP]; spriteIndex != SPRITE_INDEX_NULL; spriteIndex = nextSpriteIndex)
+        Peep* peep;
+        uint16_t sprite_index;
```

Comment:
you can just reuse the already defined `spriteIndex` instead of creating another one.